### ISSUE TITLE
Cannot read property 'normalize' of undefined

### ISSUE THREAD

**Issue by mandysimon88** [stealjs/transpile#128]
Keep hitting this error when running a very simple example. Any advice?
```
var transpile = require('transpile');
var res = transpile.to({
  name: "foo",
  source: "var GoogleMapReact = require('google-map-react')",
  metadata: { format: "es6" }
}, "cjs")
console.log(res.code)
```
=> 'Cannot read property normalize of undefined'
... goggle-map-react is an npm package, so you can try this example on your own machine for same result.

**Comment by matthewp** [comment]
Ah, sorry. There's an optional 3rd argument and it just so happens that all of our tests provide it, so this is a bug. Fix incoming soon. Thanks for finding it!

**Comment by matthewp** [comment]
This is fixed in 2.5.9 @mandysimon88 Thanks for filing the issue!

**Comment by mandysimon88** [comment]
Awesome, thank you!!
